internal/utils/version_test.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"github.com/jarcoal/httpmock"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
```

Comment:
This unit test is well designed, but the tests cases are not sufficient. It would be good to add test cases for the cases where there is no error and also with a non-empty `tag`

Reply:
Nice, just added one more, can you tell me if that is enough or should I add more?